_posts/2021-06-16-pruning-the-oldest-issues.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Pruning the oldest issues - June 2021"
+date: 2021-06-16
+author: Jan Ainali
+type: blogpost
+excerpt: Another effort to reduce our Standard for Public Code backlog
+categories:
+  - Community call
+---
+
+# Pruning the oldest issues - June 2021
+
+## Attendees
+
+* [Jan Ainali](https://publiccode.net/who-we-are/team/jan-ainali.html)
+* [Eric Herman](https://publiccode.net/who-we-are/team/eric-herman.html)
+* [Felix Faassen](https://publiccode.net/who-we-are/team/felix-faassen.html)
+
+## Log
+
+Following the [our new process](/news/2021/06/14/pruning-our-backlog.md) we continue reviewing our oldest Standard for Public Code issues. These were the ones we got through this time:
+
+- [Bundle policy and code: invite policy makers to contribute their own work in their own formats](https://github.com/publiccodenet/standard/issues/62) - session to work on this was calendered on June 14
+- [Template: Reference back to Foundations' core value propositions](https://github.com/publiccodenet/standard/issues/167) - closed after no new comments recieved since last session
+- [Add section on relation to coreinfrastrcture best-practice-badge](https://github.com/publiccodenet/standard/issues/200) - follow up in a future session
+- [Template: reorder the headers](https://github.com/publiccodenet/standard/issues/135) - closed after no new comments recieved since last session
+- [We now lay out tests for 'SHOULD' requirements](https://github.com/publiccodenet/standard/issues/155) - closed after no new comments recieved since last session
+- [Public Code can be defined more clearly](https://github.com/publiccodenet/standard/issues/203) - created [Restrict Glossary definition of 'Public Code' to concise fact](https://github.com/publiccodenet/standard/issues/502) and acknowledged that this still need to be addressed, but might be solved by [#496](https://github.com/publiccodenet/standard/issues/496)
+- [Add easier/manual ways of providing feedback to the contributor guide](https://github.com/publiccodenet/standard/issues/205) - made and merged a pull request fixing [Add the mailing list to contributing guide](https://github.com/publiccodenet/standard/issues/422) which also resolved this one
+- [Feedback: Glossary definition Code broader than use in document](https://github.com/publiccodenet/standard/issues/208) - stays open, revisit after [#496](https://github.com/publiccodenet/standard/issues/496) is resolved
+- [Do comparison on Italian national (legally enforceable) standard for Open Source](https://github.com/publiccodenet/standard/issues/209) - stays open, need work which was calendered for June 28
```

Comment:
is this work open?

```suggestion
- [Do comparison on Italian national (legally enforceable) standard for Open Source](https://github.com/publiccodenet/standard/issues/209) - stays open, needs work which was scheduled for June 28
```

Reply:
If anyone wants to join they are welcome, but we're not planning on advertising it further and making an event of it.